TLM/TLM/Manager/Impl/RoutingManager.cs

```diff
@@ -1934,7 +1934,7 @@ bool laneChangesAllowed
                                 }
 
                                 // skip lanes having lane connections
-                                if (LaneConnectionManager.Instance.HasConnections(
+                                if (LaneConnectionManager.Instance.HasOutgoingConnections(
```

Comment:
@krzychu124 in the latest commit, `HasConnections()` is faster than `HasOutgoingConnections()`. but now that the lanes are uni-directional I am not sure I can use `HasConnections()` here anymore. Unless this code path is only for roads.

What do you think? can I use `HasConnections()` here?

Reply:
Routing manager is used by the pathfinding (and DLS) to calculate paths for all vehicle types managed by TM:PE, so road vehicles, rail vehicles and soon aircraft moving around the airport. With regards to aircrafts I'm working on support for two-way taxiways (currently it's one-way, single vehicle lane) but I'll have to adjust the code anyways in that case.